src/plonkish/ir/sc.rs

```diff
@@ -114,3 +124,120 @@ impl<F: Field + Hash, MappingArgs> MappingGenerator<F, MappingArgs> {
         ctx.get_super_assignments()
     }
 }
+
+#[cfg(test)]
+mod test {
+    use halo2curves::bn256::Fr;
+
+    use super::*;
+
+    #[test]
+    fn test_default() {
+        let super_circuit: SuperCircuit<Fr, ()> = Default::default();
+
+        assert_eq!(
```

Comment:
@rutefig as the typed is ensure by rust, you are just checking that both vectors are empty in a very convoluted way, just check that super_circuit.sub_circuits.len == 0

Reply:
You marked as resolved but the change is not done

Reply:
@rutefig can you do this?